packages/build/src/telemetry/plugins.js

```diff
@@ -37,6 +42,10 @@ const enrichPayload = {
         buildVersion: pkg.version,
         // Add node version
         nodeVersion: process.version.replace(/^v/, ''),
+        // OS platform. Linix, windows, MacOS
+        osPlatform: OS_PLATFORM,
+        // OS name
+        osName: OS_NAME,
```

Comment:
What do you think about including the Linux distribution as well?
This can retrieved with the following module: https://github.com/retrohacker/getos

```js
const getos = require('getos')
const {promisify} = require('util')
const {dist: linuxDistribution} = await promisify(getos)()
// undefined if not Linux
console.log(linuxDistribution)
```